Formula/pnpm.rb

```diff
@@ -23,12 +23,39 @@ class Pnpm < Formula
   depends_on "node"
 
   def install
+    (prefix/"etc").mkpath
+    (prefix/"etc/npmrc").atomic_write "global-bin-dir = ${HOME}/Library/pnpm\n" if OS.mac?
+    (prefix/"etc/npmrc").atomic_write "global-bin-dir = ${HOME}/.local/pnpm\n" if OS.linux?
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
     bin.install_symlink Dir["#{libexec}/bin/*"]
   end
 
+  def caveats
+    pnpm_path = nil
+    on_macos do
+      pnpm_path = "$HOME/Library/pnpm"
+    end
+    on_linux do
+      pnpm_path = "$HOME/.local/pnpm"
+    end
+    <<~EOS if pnpm_path
+      Add the following to #{shell_profile} or your desired shell
+      configuration file if you would like global packages in PATH:
+        export PATH="#{pnpm_path}:$PATH"
+    EOS
+  end
+
   test do
+    pnpm_path = nil
+    pnpm_path = testpath/"Library/pnpm" if OS.mac?
+    pnpm_path = testpath/".local/pnpm" if OS.linux?
+    assert !pnpm_path.nil?, "cannot determine os"
```

Comment:
```suggestion
    pnpm_path = OS.mac? ? testpath/"Library/pnpm" : testpath/".local/pnpm"
```
Let's not worry too much about non-macOS/Linux systems for now.